File: app/src/main/java/com/github/jokar/multilanguages/MyIntentServices.java
package com.github.jokar.multilanguages;

import android.app.IntentService;
import android.content.Context;
import android.content.Intent;
import android.os.IBinder;
import android.widget.Toast;

import androidx.annotation.Nullable;

public class MyIntentServices extends IntentService {

    public MyIntentServices() {
        super("MyIntentServices");
    }


    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }

    @Override
    protected void onHandleIntent(@Nullable Intent intent) {

    }

    @Override
    public void onCreate() {
        super.onCreate();
        Toast.makeText(getApplicationContext(), getString(R.string.intent_service_create), Toast.LENGTH_SHORT).show();
    }

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
    }
}

File: app/src/main/java/com/github/jokar/multilanguages/MyIntentServices.kt
package com.github.jokar.multilanguages

import android.app.IntentService
import android.content.Intent
import android.os.IBinder
import android.widget.Toast

class MyIntentServices : IntentService("MyIntentServices") {
    override fun onBind(intent: Intent): IBinder? {
        return null
    }

    override fun onHandleIntent(intent: Intent?) {}
    override fun onCreate() {
        super.onCreate()
        Toast.makeText(applicationContext, getString(R.string.intent_service_create), Toast.LENGTH_SHORT).show()
    }

}